tinder.js

```diff
@@ -339,6 +365,28 @@ function TinderClient() {
       makeTinderCallback(callback));
   };
 
+
+  /**
+   * Post new user picture
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.uploadPicture = function(file, callback) {
+    tinderPost('image?',
```

Comment:
Sniffed out the API here, looks like the endpoint should be image?client_photo_id=ProfilePhoto{{timestamp}}, with {{timestamp}} being the current epoch time in milliseconds.